discipl/waardepapieren
# Bugfixes for icons, rendering and unwanted background scanning

This PR fixes multiple bugs
- #126 - Due to outdated dependencies the icon font wouldn't load anymore
- During the display of the scanning results the camera and QR scanner was still active in the background.
- When a long list of data is claimed the validation result indication (aka. Approved/Denied and "Afgegeven door: ....") was not visible anymore
- With the new NLX Mock data the `name` property of the configurations `SOURCE_DATA_SELECTION` is pointing to a object. This causes render/crash issues.
- Fixing reported vulnerabilities by NPM and broken tests

## Test instructions
1. Start the application and confirm the home screen matches the screenshot in the README (thus icons are correctly loaded)
2. Walk through the validation process with the BSN numer `999994669` and confirm that for "Uittreksel Basis Registratie Persoonsgegevens" and "Uittreksel Basis Regsitratie Verplaatsplaatshistorie" all the information on the paper is also visible in the app.
3. Confirm that the layout for "Uittreksel Basis Regsitratie Verplaatsplaatshistorie" matches the screenshot below:
4. Open the logs of the application (http://localhost:19002/) and point the camera of the phone to the QR code while the validation result screen is open. There should be no log activity that indicates a QR code has been scanned

![image](https://user-images.githubusercontent.com/2659866/79966582-ec6c5800-848d-11ea-9f06-572e945b891d.png)
